### ISSUE TITLE
Strokes: bug when changing cut count when stroke is not in view

### ISSUE THREAD

**Issue by vxlcoder** [CGCookie/retopoflow#1376]
to reproduce:

1. start strokes
2. stroke new geometry
3. reposition view so new geo is not in view (try turning around)
4. use redo panel or Ctrl+MouseWheel to adjust cut count

I believe this happens when reprojecting 3D stroke back to 2D.

Strokes assumes that stroke is in view

**Comment by jlampel** [comment]
RetopoFlow 3 has the same issue, so I'll put this as low priority for now, but it would be really nice to solve eventually!

**Comment by vxlcoder** [comment]
although it's not the greatest solution, this should be handled now by simply catching (and reporting to console) the exception.

at some point, it might be worth revisiting how this is handled.  (left myself a TODO)

**Comment by jlampel** [comment]
I haven't gotten an exception while doing this, even when testing before the change. I tried turning the mesh around and also panning the view so that it's fully off screen. The result was a bit messy, but no actual errors.

**Comment by vxlcoder** [comment]
i agree the current implementation produces weird results, but it's also a weird situation.  maybe sometime i will think of a good way to handle this, but it's at least not crashing :)

**Comment by jlampel** [comment]
Weird results are kinda expected here for now. Works well enough for me!
